packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * DkimAutoConfigurator is the core logic used for modals stepper (no ovh cloud domain)
+ */
+export default class DkimAutoConfigurator {
+  initializeDkimConfiguratorNoOvh() {
+    this.selector1NameInfos = '';
+    this.selector1RecordInfos = '';
+    this.selector2NameInfos = '';
+    this.selector2RecordInfos = '';
+    this.dkimSelectorsNoDomain = null;
+    this.selector1NoDomain = null;
+    this.selector2NoDomain = null;
+    this.isStepConfigureValid = false;
+    this.showConfiguratingBtn = true;
+    this.showSpfDiagnosticTitle = false;
+  }
+
+  async getSelectorNameForNoOvhCloud() {
+    if (!this.dkimSelectorsNoDomain) {
+      return;
+    }
+    const promises = this.dkimSelectorsNoDomain.map((dkimSelector) => {
+      return this.services.EmailProDomains.getDkimSelectorName(
+        this.$routerParams.organization,
```

Comment:
Line 24 should be removed since signature for EmailPro is (productId, name, selectorName)